docs/dev/toc.yml

```diff
@@ -29,6 +29,8 @@
     href: recognizers/guides/implement_recognizer.md
   - name: Implementing a Check
     href: recognizers/guides/implement_check.md
+  - name: Implementing an InstructionSet
```

Comment:
Ik zou denk ik de titel verandere naar adding recognizer to step by step mode ofzo. Omdat dat de naamgeving is die we op meerdere plekken gebruiken en instructionset meer klinkt als de losse instructies binnen de step by step